torch/onnx/symbolic_opset9.py

```diff
@@ -482,6 +482,17 @@ def prim_ConstantChunk(g, self, chunks, dim):
     return prim_ConstantSplit(g, self, split_size, dim)
 
 
+@parse_args('v', 'i', 'i')
+def unsafe_chunk(g, self, chunks, dim):
+    split_size = (self.type().sizes()[dim] + chunks - 1) // chunks
+    size = self.type().sizes()[dim]
+    splits = [split_size] * (size // split_size)
+    leftover = size % split_size
+    if leftover:
+        splits.append(leftover)
+    return g.op("Split", self, split_i=splits, axis_i=dim, outputs=1)
```

Comment:
I have instead added this `unsafe_chunk` which repeats the implementation of `prim_ConstantChunk` except that in the final statement it passes `outputs=1` instead of `outputs=len(splits)`. I think this is simpler than messing with jit pass and should still do the right thing. But it would be great if someone familiar with onnx could review my changes to this file.

Reply:
@neginraoof does that look good for you?